### ISSUE TITLE
Missing czech characters in GUI (UTF-8 problem)

### ISSUE THREAD

**Issue by evapalkova** [FoldingAtHome/fah-issues#1475]
Hi,

I created team with name "Krakonoš 12°", which is my favorite beer and it is rendered as Krakonouffffffc5uffffffa1 12uffffffc2uffffffb0. Team id is 240878. Please, how to fix this problem. I see that other special characters are not included. Please add czech alphabet or change system to UTF-8.

Thank you!
TadeasD

**Comment by ffissore** [comment]
I confirm the issue https://stats.foldingathome.org/team/240878
![immagine](https://user-images.githubusercontent.com/510465/81684083-f4009a80-9456-11ea-89d6-ad3453f719d4.png)

**Comment by shorttack** [comment]
@ffissore I agree it's a problem but there's warnings to only use English letters and numbers. Full foreign language support is a huge undertaking.

Beware: if this change is implemented, it will break a lot of third-party software and could easily fail public keys used by CURE and other teams.

I regard this as an enhancement, not a defect.

**Comment by evapalkova** [comment]
Thank you for swift answers. Yes, it would be probably a quite complex adjustment. 
This is an enhancement indeed, not a defect. However, proper charset support would enhance site attractiveness worldwide and I have the "goal of one million machines" in mind. That would be a reason for extra effort, wouldn't be?

**Comment by ffissore** [comment]
@shorttack is there a warning? I cannot see it

As for the undertaking, I agree it's big although it depends on where the problem is: if it's a wrong db encoding, then it's huge and has impacts in many points, if it's the stats webapp that it's scrambling the encoding, then it's less huge.

I would still mark this as a defect because stats shows gibberish UTF stuff: if we do not want to support non-ASCII chars, [team creation page and APIs](https://apps.foldingathome.org/team) should prevent them from being used.

Anyway, I'd leave the labelling to you, if you prefer to mark this as an enhancement

**Comment by evapalkova** [comment]
Thank you for all the effort, however, I cannot recruit more people to my team because of this issue and folding gives me no joy. 
Once it's fixed, I will come back and start folding again. Thank you for understanding. :)
